### PR TITLE
caja-customization-data: Do not build the path to the browser.xml file

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

**Review by lukefromdc — COMMENTED** [review]
Builds and runs same as master, but does NOT fix https://github.com/mate-desktop/caja/issues/1480 if it was intended to do so

File: libcaja-private/caja-customization-data.c [hunk 1]
```diff
@@ -459,45 +459,36 @@ format_name_for_display (CajaCustomizationData *data, const char* name)
...
+        while (current_node != NULL)
         {
-            /* get the category node */
-            category_node = eel_xml_get_root_child_by_name_and_property (browser_data, "category", "name", data->customization_name);
-            current_node = category_node->children;
+            char *filename, *display_name;
 
-            /* loop through the entries, adding a mapping to the hash table */
-            while (current_node != NULL)
+            display_name = xmlGetProp (current_node, "display_name");
```
Comment (raveit65) [inline libcaja-private/caja-customization-data.c:480]: We get a build warning.
```
caja-customization-data.c:480:54: warning: passing 'char [13]' to parameter of type 'const xmlChar *' (aka 'const unsigned char *') converts between pointers to integer types with different sign [-Wpointer-sign]

            display_name = xmlGetProp (current_node, "display_name");

                                                     ^~~~~~~~~~~~~~
```
https://travis-ci.org/github/mate-desktop/caja/jobs/757376330#L3473

Comment (rbuj) [reply libcaja-private/caja-customization-data.c:480]: this warning already exists on master branch, we only remove the `xml_path` variable in this PR.

Comment (raveit65) [reply libcaja-private/caja-customization-data.c:480]: OK

**Review by raveit65 — APPROVED** [review]
LGTM,
browser.xml loads fine.

**Comment by raveit65** [comment]
@lukefromdc 
Any open questions or can this be merged?

**Comment by lukefromdc** [comment]
I didn't get any new problems with this,should be good to go.

**Pull request merged**